content/post/practical-stegen.Rmd

```diff
@@ -809,15 +811,25 @@ ggplot(rr_df, aes(x = values, y = ind)) +
 The results are a lot clearer now: the tiramisu is by far the largest risk
 factor in this outbreak.
 
+# Plotting a very basic spatial overview of cases
+To complete your report, you would like to include a very basic spatial overview of cases. Based on the postal codes of all individuals with a date of symptom onset spatial coordinates were generated providing everyone with a latitude and longitude which corresponds with their household (in relation to their postal code). The variables *lat* and *long* include the longitude and latitude which we will use to plot the cases using *ggplot2*.
+
+<details>
+<summary>Advanced mapping and spatial methodologies?</b> </summary>
```

Comment:
Here, you have a closing bold element without an opening.

Reply:
<b>Apologies, messy mistake</b>